backend/mora/service/employee.py

```diff
@@ -364,6 +364,8 @@ def terminate_employee(employee_uuid):
     :param employee_uuid: The UUID of the employee to be terminated.
 
     :<json string to: When the termination should occur, as an ISO 8601 date.
+    :<json boolean full: *Optional* - perform full termination, i.e. terminate
```

Comment:
I'm not a fan of the name `full` — after, the termination of the employee is no less complete regardless of whether we terminate manager roles or merely mark them as vacant…